pydis_site/apps/content/resources/guides/python-guides/setting-different-statuses-on-your-bot.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Setting Different Statuses on Your Bot
+description: How to personalize your Discord bot status
+---
+
+You've probably seen a bot or two have a status message under their username in the member bar set to something such as `Playing Commands: .help`.
+
+This guide shows how to set such a presence, so your bot can have one as well.
```

Comment:
Better stay consistent with "status" IMO

Reply:
I know it's presence in dpy, but that's just my opinion

Reply:
I agree with this suggestion.